Automattic/wp-calypso
# Return showing duration days back

Related to `Tumblr/a8c-dsp/issues/2896`

## Proposed Changes
Previously we had the number of campaigns days and date ranges shown. Then, in 3fcd217255d07b960e163fccd5f0ce893aab7cd4 we hid this data. Now it's time to start showing it once again.

In particular, we need to show the following (as on the image below):

- duration in days: `Duration: 5 days`
- duration from-to: `Run between: Dec 5 - Dec 10`
- tooltip with time from-to: `Dec 5, 19:50 – Dec 10, 19:50`

![395144913-7c682621-1891-4365-b64b-90f483e0c8b9](https://github.com/user-attachments/assets/39202da2-ca0e-4819-ad50-8c9b65f6ae20)

Note, in addition to "Duration" and "Run between" sections, I've renamed "Clicks" to "Visitors" and "Impressions" to "People reached" as well to be in comply with the latest design from Bart.

## Testing Instructions
- For testing you need to have DSP Widget up and running locally, sandboxed WPCOM and everything we usually do in order to run Calypso with DSP Widget locally,
- In addition to above, we need to have the proper data for campaigns, so please follow instructions from `Tumblr/a8c-dsp/pull/2793`:
  - Create a campaign and change it's ID to become `110111`,
  - Import data from the mentioned PR,
  - Make sure that calling `http://localhost:3003/api/v1.1/stats/110111?start_date=2024-08-23&end_date=2024-08-27&tz=Europe/London` returns data for you,
  - Adjust start_date and end_date in DSP DB for your campaign,
- Open details for campaign `110111`, it should show data properly:

<img width="1091" alt="image" src="https://github.com/user-attachments/assets/f64ff940-f6c9-4f75-b2fb-47dc9e487e15" />

- Popover should work as well:

<img width="600" alt="image" src="https://github.com/user-attachments/assets/cfab8b82-3f22-4713-993c-126ca0dae7ed" />

## Pre-merge Checklist
